lib/pushgateway.js

```diff
@@ -63,19 +70,26 @@ function useGateway(method, job, groupings, callback) {
 		callback(err);
 	});
 
-	if(method !== 'DELETE') {
+	if (method !== 'DELETE') {
 		req.write(this.registry.metrics());
 	}
 	req.end();
 }
 
 function generateGroupings(groupings) {
-	if(!groupings) {
+	if (!groupings) {
 		return '';
 	}
-	return Object.keys(groupings).map(function(key) {
-		return ['/', encodeURIComponent(key), '/', encodeURIComponent(groupings[key])].join('');
-	}).join('');
+	return Object.keys(groupings)
+		.map(function(key) {
```

Comment:
Not so much about Prettier, but we could use templated strings instead of Array.join